in/index.md

```diff
@@ -6,29 +6,29 @@ title: Daftar Isi Bantuan
 
 <div class="panel panel-info">
   <div class="panel-heading">
-    <strong>You can't find a solution to your problem? You still have questions?</strong>
```

Comment:
This is currently not translatable. It is too hard for us to adapt https://github.com/JabRef/help.jabref.org/blob/gh-pages/_scripts/automate.py accordingly.

The templates are residing in https://github.com/JabRef/help.jabref.org/tree/gh-pages/_includes

Can you please translate https://github.com/JabRef/help.jabref.org/blob/gh-pages/_includes/link-to-toc-in.html and https://github.com/JabRef/help.jabref.org/blob/gh-pages/_includes/link-to-main-in.html?

Reply:
I'll try to translate it